core/src/main/scala/cats/Applicative.scala

```diff
@@ -29,6 +30,12 @@ import simulacrum.typeclass
   def pureEval[A](x: Eval[A]): F[A] = pure(x.value)
 
   /**
+    * Given fa and n, apply fa n times to construct an F[List[A]] value.
```

Comment:
This might render a little cleaner in the ScalaDoc if you use backticks around `fa`, `n`, and `F[List[A]]`.